StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentSheetFlowController.swift

```diff
@@ -81,6 +81,28 @@ extension PaymentSheet {
             return viewController.selectedPaymentOption
         }
 
+        // Stores contextual state releated to the update API
+        private var latestUpdateContext: UpdateContext?
+
+        struct UpdateContext {
+            /// The ID of the last update API call
```

Comment:
```suggestion
            /// The ID of the update API call
```
this is a nitpick, but the object doesn't know whether it is the last update or not - it just represents an 'update'.  The user decides whether it is going to use this object to represent the last update, or maintain a queue of updates, etc.